src/main/kotlin/net/rrerr/minetrix/MatrixRoomMessageReceivedListener.kt

```diff
@@ -82,4 +87,15 @@ class MatrixRoomMessageReceivedListener(private val plugin: Main) : RoomMessageR
 
     private fun getMediaUrl(message: UrlMessageContent) =
         "${plugin.matrixClient!!.hs.base}/_matrix/media/v1/download/${message.url!!.toString().replace("mxc://", "")}"
+
+    @Throws(NoTokenException::class)
+    private fun sendReadMarker(event: RoomMessageReceivedEvent) {
+        val eventId = event.id
+        val id = plugin.room!!.id
+        val parser = Parser.default()
+        val body = parser.parse(StringBuilder("{\"m.read\": \"$eventId\"}")) as JsonObject
```

Comment:
According to the [spec you linked](https://matrix.org/docs/spec/client_server/latest#post-matrix-client-r0-rooms-roomid-read-markers), the parameter `m.fully_read` is required whereas `m.read` is optional.  Did you mean to use the former here?